string which_camera
---
sensor_msgs/Image image
